steambox_nvidia/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM steambox
+MAINTAINER Jereme Corrado "jereme@zoion.net"
+
+USER root
+
+RUN apt-get install -y debian-archive-keyring
+RUN echo "deb http://ftp.debian.org/debian/ jessie-backports main contrib non-free" > /etc/apt/sources.list.d/jessie-backports.list
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y -t jessie-backports nvidia-driver
+
```

Comment:
@jcorrado   If I understand this correctly, it will pull the backported Debian Jessie nvidia driver into the SteamOS tree.

My understanding is that the SteamOS drivers were "optimized for games", which is one of the big selling points of running SteamOS -- improved game performance.

OTOH, I've also read that the container driver must match the host driver (and I understand your host is true Debian).

My solution to this was to install the ATI-proprietary driver in both container and host, using DKMS.  I wonder if you'd see better game compatibility/performance if you do something similar?

BTW, here is a thread I found for installation of the proprietary driver.  It is definitely more steps, and I'm not sure if it can be done "unattended".  (The "unattended" bit is the reason there are two dockerfiles in `steambox_ati`.)  Let me know your thoughts.
   http://steamcommunity.com/groups/steamuniverse/discussions/1/540743757298243719/

As with the other comment, I'll merge what you tell me is best -- I have no Nvidia machine of my own.  I'm just appreciative that you're doing the work. :)